src-docs/src/index.html

```diff
@@ -14,7 +14,11 @@
     <meta property="og:image" content="https://repository-images.githubusercontent.com/107422373/b6180480-a1d7-11eb-8a3c-902086232aa7">
     <meta property="og:url" content="hhttps://elastic.github.io/eui">
     <meta name="twitter:card" content="summary_large_image">
+    <meta name="emotion-styles-global">
+    <meta name="emotion-styles-eui">
+    <meta name="emotion-styles-compiled">
```

Comment:
`emotion-styles-compiled` was super confusing to me at first glance - aren't these compiled **Sass** styles?
```suggestion
    <meta name="sass-styles-compiled">
```